### DISCUSSION TITLE
How to Make my Contributions Show on my Heat Map.

### DISCUSSION THREAD

**Discussion by Casmir293** [community/community#109236] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How can I make my GitHub contributions to repositories owned by organizations appear on my personal contribution graph?

I joined a team and I have been making contributions ever since but it doesn't get indicated on my GitHub activity graph.

**Comment by Shahnawazk7944** [comment]
Hey @Casmir293  
The two articles will answer all your questions.

**Viewing contributions on your profile:**
 https://docs.github.com/articles/viewing-contributions-on-your-profile-page

**Why are my contributions not showing up on my profile?:** 
https://docs.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile

**Reply by Casmir293** [reply]
I have done all of these. I have been making commits on the project's development branch and have yet to make a Pull Request. If I make the pull request and it gets merged to the main branch will it make the commits show on the heat map?

**Reply by Shahnawazk7944** [reply]
Unfortunately, merging a pull request to the main branch won't necessarily make your commits show on the GitHub heatmap if they were originally made on the development branch. GitHub's heatmap primarily tracks "pushes" to the main branch, rather than individual commits merged through pull requests. This means that even if your pull request gets merged, the individual commits from the development branch might not be reflected in the heatmap.

**Reply by Casmir293** [reply]
Thanks for your feedback 💯

**Reply by Shahnawazk7944** [reply]
You're welcome. It's all about sharing my knowledge. There may be some exceptions, so try looking for solutions on the internet.

**Comment by rirefat** [comment]


**Reply by Casmir293** [reply]
I have done all of these. I have been making commits on the project's development branch and have yet to make a Pull Request. If I make the pull request and it gets merged to the main branch will it make the commits show on the heat map?
